packages/loader/container-loader/src/containerContext.ts

```diff
@@ -211,8 +211,8 @@ export class ContainerContext implements IContainerContext {
         return this.container.attachState;
     }
 
-    public createSummary(): ISummaryTree {
-        return this.runtime.createSummary();
+    public createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree {
```

Comment:
Can you please add comment (ideally here and in definitions) on the expectations and meaning of the argument?